infrastructure/scripts/aws/run_tests.sh

```diff
@@ -39,11 +39,17 @@ echo "HOSTS=${HOSTS}"
 ssh ${SSH_OPTIONS} geode@$FIRST_INSTANCE "\
   rm -rf geode-benchmarks geode && \
   git clone https://github.com/apache/geode geode && \
-  (pushd geode; git checkout ${BRANCH}) && \
-  (pushd geode; ./gradlew pTML -PversionNumber=${DATE} -PreleaseType="-BENCHMARKBUILD") && \
+  (pushd geode && git checkout ${BRANCH}) && \
+  (pushd geode && ./gradlew install installDist)"
+
+
+VERSION=$(ssh ${SSH_OPTIONS} geode@$FIRST_INSTANCE geode/geode-assembly/build/install/apache-geode/bin/gfsh version)
```

Comment:
Is there a reason why this is done in 3 ssh sessions to one machine rather than one? 
[export VERSION="$(./gfsh version)"]

Reply:
```
ssh ${SSH_OPTIONS} geode@$FIRST_INSTANCE << EOF
  rm -rf geode-benchmarks geode
  git clone https://github.com/apache/geode geode
  (pushd geode && git checkout ${BRANCH})
  (pushd geode && ./gradlew install installDist)

  export VERSION="$(geode/geode-assembly/build/install/apache-geode/bin/gfsh version)"

  git clone https://github.com/apache/geode-benchmarks --branch ${BENCHMARK_BRANCH} && \
  cd geode-benchmarks && \
  ./gradlew -PgeodeVersion=$VERSION benchmark -Phosts=${HOSTS}"
EOF
```

Reply:
Go for it!

Reply:
I don't if this has any downfalls. @smgoller can you comment ? I remember you changed my EOF way of writing ssh scripts to '\' method.